### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: __unported__/purchase_payment/__openerp__.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "version" : "1.0",
     "author" : "Pexego",
     "website": "www.pexego.es",
+    "license": "AGPL-3",
     "license" : "GPL-3",
```
Comment (StefanRijnhart) [inline __unported__/purchase_payment/__openerp__.py]: Double key here. Also note that you need permission to relicense. Having a GPL module in the OCA is not problematic anyway.

File: __unported__/pxgo_bank_statement_running_balance/__openerp__.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
         "version" : "1.0",
         "author" : "Pexego for Igalia (http://www.igalia.com/)",
         "website" : "http://www.pexego.es",
+    "license": "AGPL-3",
```
Comment (StefanRijnhart) [inline __unported__/pxgo_bank_statement_running_balance/__openerp__.py]: Odd indentation here and elsewhere.

**Comment by yvaucher** [comment]
@StefanRijnhart Fixed thanks for the review

**Comment by StefanRijnhart** [comment]
Tnx! :+1:

**Comment by lmignon** [comment]
:+1:  Unrelated travis error

**Pull request merged**